frontend/alanda/libs/common/src/lib/validators/commentRequiredValidator.ts

```diff
@@ -1,5 +1,9 @@
 import { AbstractControl, ValidationErrors, ValidatorFn } from '@angular/forms';
 
+interface HasValue{
+  readonly value: any;
+}
+
```

Comment:
why an interface with value? instead of declaring value: any as parameter for the function? or a callback function which determines the value?